app/src/main/java/com/dcrandroid/activities/ProposalDetailsActivity.kt

```diff
@@ -77,7 +82,8 @@ class ProposalDetailsActivity : BaseActivity() {
 
         // load file from server if it is not yet loaded or outdated.
         if (proposal.indexFile!!.isNotEmpty() && proposal.indexFileVersion == proposal.version) {
-            proposal_description.text = proposal.indexFile
+            // set markdown
+            markwon.setMarkdown(proposal_description, proposal.indexFile!!)
```

Comment:
```suggestion
            Markwon.create(this)
                    .setMarkdown(proposal_description, proposal.indexFile!!)
```
I don't think a global variable is required. If you need to reuse the variable then declare it global `loadProposalDetails()`.